### PR TITLE
Add getSlotsAvailable function

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "thinkful-ui",
-  "version": "1.0.3",
+  "version": "1.0.6",
```
Comment (learykara) [inline package.json]: why go from 1.0.3 to 1.0.6?

File: src/AvailabilityGrid/AvailabilityGrid.jsx [hunk 2]
```diff
@@ -116,11 +116,11 @@ const AvailabilityGrid = React.createClass({
       formatString = 'h:mma';
     }
 
-    slotNames = Array.fill(new Array(SLOTS_DAY)).map((slot, index) => (
-        moment(moment().format('YYYY-MM-DD')).
-        add(MINUTES_SLOT * (index + 1), 'minutes').
-        format(formatString)
-    ));
+    let currentSlot = moment(moment().format('YYYY-MM-DD'))
```
Comment (learykara) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: what? why not just `let currentSlot = moment()` ?

File: src/AvailabilityGrid/AvailabilityGrid.jsx [hunk 3]
```diff
@@ -211,6 +211,14 @@ const AvailabilityGrid = React.createClass({
     }).join('');
   },
 
+  getSlotsAvailable() {
+    return _.sum(this.state.days.map((dayData) => {
```
Comment (learykara) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: don't need parens when there's just one arg

File: src/AvailabilityGrid/AvailabilityGrid.jsx [hunk 4]
```diff
@@ -116,11 +116,11 @@ const AvailabilityGrid = React.createClass({
...
 
-    slotNames = Array.fill(new Array(SLOTS_DAY)).map((slot, index) => (
-        moment(moment().format('YYYY-MM-DD')).
-        add(MINUTES_SLOT * (index + 1), 'minutes').
-        format(formatString)
-    ));
+    let currentSlot = moment(moment().format('YYYY-MM-DD'))
+    for (let i = 0; i < SLOTS_DAY; i++) {
+      slotNames.push(currentSlot.format(formatString));
+      currentSlot.add(MINUTES_SLOT, 'm');
```
Comment (learykara) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: does this update `currentSlot` in place?

**Comment by learykara** [comment]
couple of comments, otherwise :+1: .  let me know once it's merged and stuff and I'll publish.

File: package.json (hunk 1 above)
Comment (rkuykendall) [inline package.json]: Develop is up to 1.0.5. This branch was on 1.0.3 though.

File: src/AvailabilityGrid/AvailabilityGrid.jsx (hunk 2 above)
Comment (rkuykendall) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: It needs to be the very beginning of the day. `moment()` is now.

File: src/AvailabilityGrid/AvailabilityGrid.jsx (hunk 3 above)
Comment (rkuykendall) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: Thanks!

File: src/AvailabilityGrid/AvailabilityGrid.jsx (hunk 4 above)
Comment (rkuykendall) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: `add` It increments it by a number of minutes, so it's slowly working it's way up the timeline.

**Comment by rkuykendall** [comment]
@learykara Responded to a few questions, made change suggested.

File: src/AvailabilityGrid/AvailabilityGrid.jsx (hunk 2 above)
Comment (learykara) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: you can do `moment().startOf('day')`

Comment (benjaminEwhite) [inline src/AvailabilityGrid/AvailabilityGrid.jsx]: beat me to it @learykara

**Pull request merged**